src/frontend/js/widgets/UserLogin/index.spec.tsx

```diff
@@ -30,30 +33,50 @@ jest.mock('utils/indirection/window', () => ({
 
 jest.mock('utils/context', () => ({
   __esModule: true,
-  default: mockRichieContextFactory().one(),
+  default: mockRichieContextFactory({
+    authentication: {
+      backend: 'fonzie',
+      endpoint: 'https://auth.local.test',
+    },
+    joanie_backend: {
+      endpoint: 'https://endpoint.test',
+    },
+  }).one(),
 }));
 
 describe('<UserLogin />', () => {
+  const Wrapper = ({
+    children,
+    user = false,
+  }: PropsWithChildren & { user?: Nullable<User | boolean> }) => (
+    <QueryClientProvider client={createTestQueryClient({ user })}>
+      <IntlProvider locale="en">
+        <JoanieSessionProvider>{children}</JoanieSessionProvider>
+      </IntlProvider>
+    </QueryClientProvider>
+  );
+  beforeEach(() => {
+    fetchMock
+      .get('https://endpoint.test/api/v1.0/orders/', [])
+      .get('https://endpoint.test/api/v1.0/addresses/', [])
+      .get('https://endpoint.test/api/v1.0/credit-cards/', []);
+  });
   afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
We do not mock new things, I do not understand why we have to add this `jest.clearAllMocks()`